src/visitors/Tag.js

```diff
@@ -54,8 +69,19 @@ function getAttributes(
 
       const expr = parseExpression(val === true ? 'true' : val, context);
 
-      if (!mustEscape && (!t.isStringLiteral(expr) || /(\<\>\&)/.test(val))) {
-        throw new Error('Unescaped attributes are not supported in react-pug');
+      if (!mustEscape) {
+        const isStringViaAliases: boolean =
+          t.isStringLiteral(expr) && !['className', 'id'].includes(name);
```

Comment:
For a list of two, I think I'd prefer just `(name === 'className' || name === 'id')` - it takes less time to mentally parse.